File: Python1/Calculadora.py
numero1 = int(input("Primer numero"))
numero2 = int(input("Segundo numero"))
operacion = input("¿Qué operación quieres realizar? (+, -, * o /)")

match operacion:
    case "+":
        total = numero1 + numero2 
    case "-":
        total = numero1 - numero2
    case "*":
        total = numero1 * numero2
    case "/":
        if numero2 == 0:
            total = "Operación no se puede realizar. División por 0"
        else:
            total = numero1 / numero2
    case _:
        total = "Operación no soportada"

print(f"Resultado:  {total}")

File: EJ1/Calculadora.php
<?php
    
$numero1 = readline("Introduce el primer numero: ");
$numero2 = readline("Introduce el segundo numero: ");
$operacion = readline("Introduce la operacion a realizar (+, -, *, /, %, **)");

switch ($operacion) {
    case "+":
        $total = $numero1 + $numero2;
        echo "El total es de = " . $total;
        break;
    case "-":
        $total = $numero1 - $numero2;
        echo "El total es de = " . $total;
        break;
    case "*":
        $total = $numero1 * $numero2;
        echo "El total es de = " . $total;
        break;
    case "/":
        if ($numero2 != 0) {
            $total = $numero1 / $numero2;
            echo "El total es de = " . $total;
            break;
        } else {
            echo "No se puede dividir por cero";
            break;
        }
    case "%":
        $total = $numero1 % $numero2;
        echo "El total es de = " . $total;
        break;
    case "**":
        $total = $numero1 ** $numero2;
        echo "El total es de = " . $total;
        break;
}
?>